process/smartContract/process_test.go

```diff
@@ -3270,7 +3275,8 @@ func TestScProcessor_ProcessSmartContractResultExecuteSCIfMetaAndBuiltIn(t *test
 	require.True(t, executeCalled)
 
 	executeCalled = false
-	_ = sc.flagBuiltInFunctionOnMetachain.SetReturningPrevious()
+	enableEpochsHandlerStub.IsBuiltInFunctionOnMetaFlagEnabledField = true
+	enableEpochsHandlerStub.IsBuiltInFunctionsFlagEnabledField = true
```

Comment:
also here

Reply:
needed to complete the execution of the builtin function [here](https://github.com/ElrondNetwork/elrond-go/blob/integrate_enable_epochs_handler_p2/process/smartContract/process.go#L846)